pkg/utils/k8sutil.go

```diff
@@ -453,21 +453,33 @@ func CreateK8sCustomResource(runtime, handler, file, funcName, funcType, topic,
 		Do().Into(&f)
 	if err != nil {
 		if k8sErrors.IsNotFound(err) {
+			funcLabels := map[string]string{}
+			for _, label := range labels {
+				k, v := getKV(label)
+				funcLabels[k] = v
+			}
+
+			funcAnnotation := map[string]string{
+				"Description": description,
+			}
 			f := &spec.Function{
 				TypeMeta: metav1.TypeMeta{
 					Kind:       "Function",
 					APIVersion: "k8s.io/v1",
 				},
 				Metadata: metav1.ObjectMeta{
-					Name:      funcName,
-					Namespace: ns,
+					Name:        funcName,
+					Namespace:   ns,
+					Labels:      funcLabels,
+					Annotations: funcAnnotation,
 				},
 				Spec: spec.FunctionSpec{
 					Handler:  handler,
 					Runtime:  runtime,
 					Type:     funcType,
 					Function: readFile(file),
 					Topic:    topic,
+					Env:      envs,
```

Comment:
See comment above about using `getKV` for labels but not for envs.  Should we be using `getKV` here too?  If not, you need to specify how Env is going to be interpreted on the server-side.